test/harry/main/org/apache/cassandra/harry/model/BytesPartitionState.java

```diff
@@ -135,7 +141,13 @@ private long[] toDescriptor(ImmutableUniqueList<Symbol> positions, Map<Symbol, B
             Symbol column = positions.get(i);
             if (values.containsKey(column))
             {
-                long vd = factory.valueCache.deflate(new Value(column.type(), values.get(column)));
+                ByteBuffer value = values.get(column);
+                if (value == null || !value.hasRemaining() && (column.type().isUDT() && column.type().isMultiCell()))
```

Comment:
user type is the only multi cell type i see that allows `<empty>` so this check should be fine; can expand if we find more cases